src/components/UI/InputAndSelectCombined/styles.ts

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+/* eslint-disable-next-line */
```

Comment:
why here?))

Reply:
prefer default export. Changed to more specific `/* eslint-disable import/prefer-default-export */`

Reply:
I think it is redundant eslint setting